### ISSUE TITLE
Region handling: some valid ds9 regions are not supported

### ISSUE THREAD

**Issue by keflavich** [CARTAvis/carta#71]
I encountered an issue where a region file formatted like this:
```
icrs
point(17:25:26.34, -28:30:40.54) # point=x
```
is not recognized as a region file by CARTA, but it is a valid ds9 region file.

**Comment by kswang1029** [comment]
We had an issue due to the dos format. Now fixed in dev https://github.com/CARTAvis/carta-backend/pull/658

**Comment by kswang1029** [comment]
Do you have the header line starting with #?

**Comment by keflavich** [comment]
no, there is no header line

**Comment by kswang1029** [comment]
please add

`# Region file format: DS9 CARTA 1.4`

as the 1st line, otherwise it is hard to tell the region file is in ds9 format or crtf.

**Comment by keflavich** [comment]
I know that's a workaround, but that is not a requirement for a file to be a valid ds9 region file.  The user should be able to tell CARTA what type of region file it is and try to read it.  This is important because there are potentially enormous numbers of ds9-compatible .reg files around, and if CARTA can only read a subset of them, it results in creation of a new format standard.

**Comment by veggiesaurus** [comment]
> I know that's a workaround, but that is not a requirement for a file to be a valid ds9 region file. The user should be able to tell CARTA what type of region file it is and try to read it. This is important because there are potentially enormous numbers of ds9-compatible .reg files around, and if CARTA can only read a subset of them, it results in creation of a new format standard.

I agree. CRTF requires `#CRTF` in the first line, so that narrows things down, and we can attempt to parse things as DS9 if they're missing that header line

**Comment by keflavich** [comment]
iirc, ds9 provided a large suite of test files, but I can't find it.  In astropy-regions, we have a bunch too, though all of these presently have headers - this issue motivates adding a headerless test case:
https://github.com/astropy/regions/tree/main/regions/io/ds9/tests/data

**Comment by keflavich** [comment]
ah, the `.strip.reg` files are headerless

**Comment by o-smirnov** [comment]
Experiencing a similar problem with DS9 line regions:

![image](https://user-images.githubusercontent.com/6470079/128869839-11283d58-3b63-4d3b-a372-aa47e7417fd6.png)

Region file: [im2di_clustered.tessel.reg.txt](https://github.com/CARTAvis/carta/files/6961783/im2di_clustered.tessel.reg.txt)
